core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.ts

```diff
@@ -332,10 +383,59 @@ angular.module('oppia').directive('stateTranslation', [
             return {'border-left': '3px solid ' + color};
           };
 
-          $scope.getHtmlSummary = function(subtitledHtml) {
-            var htmlAsPlainText = $filter(
-              'formatRtePreview')(subtitledHtml.getHtml());
-            return htmlAsPlainText;
+          $scope.getSubtitledContentSummary = function(subtitledContent) {
+            if (subtitledContent instanceof SubtitledHtml) {
+              return $filter('formatRtePreview')(subtitledContent.getHtml());
+            } else if (subtitledContent instanceof SubtitledUnicode) {
+              return subtitledContent.getUnicode();
+            }
+          };
+
+          $scope.getInteractionCustomizationArgTranslatableContent = function(
+              customizationArgs: InteractionCustomizationArgs
+          ): {name: string, content: SubtitledUnicode|SubtitledHtml}[] {
+            const translatableContent = [];
+
+            const camelCaseToSentenceCase = (s) => {
```

Comment:
Are we expecting this method to handle UpperCameCase?

Reply:
It probably isn't necessary, as the customization argument names are all lower camel case, but I've added logic to handle this- thanks!

Reply:
I asked this as I thought it will return extra space for string stats with capital letter (not sure!)* (Either add a comment in the fileoverview that this only works for lower camel case or if you want you can fix it!)